pytorch_lightning/accelerators/ddp_backend.py

```diff
@@ -145,17 +170,7 @@ def set_world_ranks(self, process_idx):
         self.trainer.world_size = self.trainer.num_nodes * self.trainer.num_processes
 
     def model_to_device(self, model, process_idx, is_master):
```

Comment:
is_master isn't used and can be removed throughout the file

Reply:
done